terraform/deployments/cluster-services/argo.tf

```diff
@@ -101,6 +101,17 @@ resource "helm_release" "argo_workflows" {
   values = [yamlencode({
     controller = {
       workflowNamespaces = concat([local.services_ns], var.argo_workflow_namespaces)
+      workflowDefaults = {
+        spec = {
+          activeDeadlineSeconds = 7200
```

Comment:
This is a tricky one. This timeout will be unexpected. 2 hours seems fine, easy to change later.

Reply:
yeah, I was thinking that most jobs run at most for 1 hour or less afaik. Maybe we can investigate further or add the warning in the docs.